### PR TITLE
Equal and NotEqual converters

### PR DESCRIPTION
**Pull request by bijington**
Documentation added for .NET MAUI Community Toolkit proposals:

- `[Proposal] EqualConverter` <https://github.com/CommunityToolkit/Maui/issues/60>
- `[Proposal] NotEqualConverter` <https://github.com/CommunityToolkit/Maui/issues/73>

### CODE REVIEW

**Comment by PRMerger19** [comment]
## PRMerger Results
 | Issue | Description |
 --- | --- |
 Added File(s) | This PR contains added files. New files require human review.
 File Change Percent | This PR contains file(s) with more than 20% file change.

**Review by brminnick — CHANGES_REQUESTED** [review]
Thanks Shaun! Just a couple small tweaks 👍

**Comment by bijington** [comment]
> Thanks Shaun! Just a couple small tweaks 👍 

Thanks. This really wasn't my finest piece of work 🤦

**Comment by PRMerger5** [comment]
## PRMerger Results
 | Issue | Description |
 --- | --- |
 Added File(s) | This PR contains added files. New files require human review.
 File Change Percent | This PR contains file(s) with more than 20% file change.

File: docs/maui/converters/not-equal-converter.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+        var label = new Label { Text = "The value is not equal to 100" };
+
+		label.SetBinding(
+			Label.IsVisibleProperty,
+			new Binding(
+				nameof(ViewModels.MyValue),
+				converter: new EqualConverter(),
+				converterParameter: 100));
+
+		Content = label;
```
Comment (bijington) [inline docs/maui/converters/not-equal-converter.md:61]: @brminnick how does this render on the docs page? The indentation looks fine in VS Code on my machine but I wonder if I need to tweak the settings

**Review by brminnick — CHANGES_REQUESTED** [review]
image

Shoot - I just noticed these converters are missing from the TOC

**Comment by PRMerger20** [comment]
## PRMerger Results
 | Issue | Description |
 --- | --- |
 Added File(s) | This PR contains added files. New files require human review.
 Yaml File(s) | This PR includes changes to .yml file(s) owned by another author.
 File Change Percent | This PR contains file(s) with more than 20% file change.

**Review by brminnick — APPROVED** [review]

**Pull request merged**